src/main/java/com/recordit/server/service/MemberService.java

```diff
@@ -109,4 +109,15 @@ public Long modifyMember(ModifyMemberRequestDto modifyMemberRequestDto) {
 
 		return member.modify(modifyMemberRequestDto);
 	}
+
+	public Long logout() {
+		Long userIdBySession = sessionUtil.findUserIdBySession();
+
+		Member member = memberRepository.findById(userIdBySession)
+				.orElseThrow(() -> new MemberNotFoundException("회원 정보를 찾을 수 없습니다."));
+
+		sessionUtil.invalidateSession();
+
+		return userIdBySession;
+	}
```

Comment:
```suggestion
	public void logout() {
		sessionUtil.invalidateSession();
	}
```
요롷게 바꾸는건 어떨까요? 회원정보를 찾을 수 없는 경우는 사용자가 세션을 들고있으나 해당세션이 redis에 없는 경우인데,
굳이 DB에 호출을 해야하는지가 의문이 들어 이렇게 생각했는데 어떠세용?

Reply:
네, 도모 말대로  굳이 DB조회를 안해도 될거같아요!